test/install_test.go

```diff
@@ -235,7 +235,7 @@ func TestInstallOrUpgradeCli(t *testing.T) {
 		}
 
 		// prepare for stage 2
-		args = append([]string{"control-plane"}, args...)
+		args = append([]string{"control-plane", "--addon-overwrite"}, args...)
```

Comment:
I think we shouldn't add this flag as it doesn't reflect the usual upgrade scenario. Without it the upgrade fails from edge-20.5.3 like described in #4471, but it's green without it using the current latest edge.

Reply:
I don't think this works for any edge right now. I ran into this issue when upgrading from `edge-20.6.1` as well. I originally opened an issue for it, but closed it (#4556).

Reply:
Ah sorry, I was testing in a different branch... I confirmed it's failing upgrading from Friday's edge as well. From what I can see, the resulting upgrades with and without `--from-manifests` are different. The former isn't touching Grafana's version while the latter is. @Pothulapati can you lend a hand to see what's going on?

Reply:
This was updated to get `configmaps/linkerd-config-addons` if it exists.